README.md

```diff
@@ -78,6 +78,8 @@ This will allow you to run your favourite IDE but not have to install any depend
 #### Prerequisites
 
 - [Docker](https://www.docker.com/)
+- [Documentation](https://docs.docker.com/config/containers/resource_constraints/)
+- [Docker Desktop](https://www.docker.com/products/docker-desktop/)
 - [Docker Compose](https://github.com/docker/compose) V2.
```

Comment:
[Proofreading] I think adding 'V2' after the name of the docker would make it easier for contributors to understand which version you're referring to. 
```suggestion
- [Docker Compose V2](https://github.com/docker/compose) V2
```

Reply:
This results in there being x2 `V2` as shown on the screenshot so this will not work. 
<img width="286" alt="Screenshot 2023-05-24 at 10 57 55" src="https://github.com/EddieHubCommunity/LinkFree/assets/82668196/d40818fe-9fb8-4db5-8326-dc9052876554">